pkg/utils/utils.go

```diff
@@ -98,8 +103,12 @@ func BuildCSPDirectives(k8sMode, pluginsCSP, indexPageScriptNonce string) ([]str
 				scriptSrcDirective = append(scriptSrcDirective, sources...)
 			case consolev1.StyleSrc:
 				styleSrcDirective = append(styleSrcDirective, sources...)
+			case consolev1.ObjectSrc:
+				objectSrcDirective = append(objectSrcDirective, sources...)
+			case consolev1.ConnectSrc:
+				connectSrcDirective = append(connectSrcDirective, sources...)
 			default:
-				klog.Warningf("ignored invalid CSP directive: %v", directive)
+				klog.Infof("ignored invalid CSP directive: %v", directive)
```

Comment:
nit: This isn't necessarily an invalid directive since there are many we don't allow to be set

```suggestion
				klog.Infof("ignored unsupported CSP directive: %v", directive)
```